### ISSUE TITLE
Deploy to Production

### ISSUE THREAD

**Issue by rajadain** [project-icp/bee-pollinator-app#530]
Once the new fixes are in, deploy to production.

Notes:

- [ ] Run migrations from #533

Depends on #531 
Depends on #532

**Comment by fungjj92** [comment]
Both issues are merged into develop, but dependabot made a bunch of security fix PRs. I will review & merge those for this release, too.

**Comment by fungjj92** [comment]
All desired PR's are merged into the release branch. The deploy is good to go.
